scheme/parsec-cca/evidence_handler.go

```diff
@@ -0,0 +1,418 @@
+	}
+
+	implID, err = common.GetMandatoryPathSegment("PARSEC_CCA.impl-id", fields)
+	if err != nil {
+		return nil, fmt.Errorf("unable to synthesize reference value abs-path: %w", err)
+	}
+
+	lookupKey := arm.RefValLookupKey(SchemeName, tenantID, implID)
+	log.Debug("PARSEC CCA Plugin Reference Value Look Up Key= %s\n", lookupKey)
+
+	return []string{lookupKey}, nil
+}
+
+func (s EvidenceHandler) SynthKeysFromTrustAnchor(tenantID string, ta *proto.Endorsement) ([]string, error) {
+	var (
+		instID string
+		implID string
+		fields map[string]*structpb.Value
+		err    error
+	)
+
+	fields, err = common.GetFieldsFromParts(ta.GetAttributes())
+	if err != nil {
+		return nil, fmt.Errorf("unable to synthesize trust anchor abs-path: %w", err)
+	}
+
+	implID, err = common.GetMandatoryPathSegment("PARSEC_CCA.impl-id", fields)
+	if err != nil {
+		return nil, fmt.Errorf("unable to synthesize trust anchor abs-path: %w", err)
+	}
+
+	instID, err = common.GetMandatoryPathSegment("PARSEC_CCA.inst-id", fields)
+	if err != nil {
+		return nil, fmt.Errorf("unable to synthesize trust anchor abs-path: %w", err)
+	}
+
+	lookupKey := arm.TaLookupKey(SchemeName, tenantID, implID, instID)
+	log.Debug("PARSEC CCA Plugin TA Look Up Key= %s\n", lookupKey)
+	return []string{lookupKey}, nil
+}
+
+func (s EvidenceHandler) GetTrustAnchorID(token *proto.AttestationToken) (string, error) {
+	var parsecCca cca.Evidence
+
+	err := parsecCca.FromCBOR(token.Data)
+	if err != nil {
+		return "", handler.BadEvidence(err)
+	}
+
+	return arm.TaLookupKey(
+		SchemeName,
+		token.TenantId,
+		arm.MustImplIDString(parsecCca.Pat.PlatformClaims),
+		arm.MustInstIDString(parsecCca.Pat.PlatformClaims),
+	), nil
+
+}
+
+func (s EvidenceHandler) ExtractClaims(token *proto.AttestationToken, trustAnchor string) (*handler.ExtractedClaims, error) {
+	var (
+		extracted handler.ExtractedClaims
+		parsecCca cca.Evidence
+		claimsSet = make(map[string]interface{})
+		kat       = make(map[string]interface{})
+	)
+
+	if err := parsecCca.FromCBOR(token.Data); err != nil {
+		return nil, handler.BadEvidence(err)
+	}
+	kat["nonce"] = *parsecCca.Kat.Nonce
+	key := parsecCca.Kat.Cnf.COSEKey
+	jkey, err := json.Marshal(key)
+	if err != nil {
+		return nil, handler.BadEvidence(err)
+	}
+	// Stash it as a string otherwise structpb will marshal it again
+	kat["akpub"] = string(jkey)
+
+	claimsSet["kat"] = kat
+	pmap, err := common.ClaimsToMap(parsecCca.Pat.PlatformClaims)
+	if err != nil {
+		return nil, handler.BadEvidence(err)
+	}
+	claimsSet["cca.platform"] = pmap
+	rmap, err := common.ClaimsToMap(parsecCca.Pat.RealmClaims)
+	if err != nil {
+		return nil, handler.BadEvidence(err)
+	}
+	claimsSet["cca.realm"] = rmap
+
+	extracted.ClaimsSet = claimsSet
+
+	extracted.ReferenceID = arm.RefValLookupKey(
+		SchemeName,
+		token.TenantId,
+		arm.MustImplIDString(parsecCca.Pat.PlatformClaims),
+	)
+	log.Debug("extracted Reference ID Key = %s", extracted.ReferenceID)
```

Comment:
ditto (use the formatting version)

Reply:
this has not been addressed

Reply:
Thanks, 
Edited Issue #173 to handle this case and noticed a similar correction required in plugin for `cca-ssd-platform`.

Both plugins will be addressed by the above issue.